content/ADR-125.md

```diff
@@ -0,0 +1,208 @@
+---
+adr: 125
+title: User interface components for SDK 7
+date: 2022-11-14
+status: Living
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - menduz
+---
+
+## Abstract
+
+<!--
+Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the document section. **Someone should be able to read only the abstract to get the gist of what this document is about in its current state.** Abstracts should be always up to date with the current state of the document.
+-->
+
+This specification covers an initial set of components of the SDK to create flexbox-based user interfaces. The components only describe behavior and configuration skipping position and sizing, that is considered layouting and is given by the UiTransform component from [ADR-124](/adr/ADR-124).
+
+## Specification
+
+<!--
+The technical specification should describe the syntax and semantics of any new feature.
+-->
+
+### `PointerEvents` component
+
+We are going to reuse the existing PointerHoverFeedback component, the name will be changed to PointerEvents to reflect its ultimate purpose.
+
+The `PointerEvents` component is used to signal the renderer about which entities are eligible for pointer interactions, it is RECOMMENDED to use this component as filter for raycasts.
+
+```protobuf
+enum PointerEventType {
+  PET_UP = 0;
+  PET_DOWN = 1;
+  PET_HOVER_ENTER = 2;
+  PET_HOVER_LEAVE = 3;
+}
+
+// We can talk about this name 
+message PBPointerEvents {
+  message Info {
+    optional common.InputAction button = 1; // default=InputAction.ANY
```

Comment:
Some uses of `common` are prefixed by `decentraland.` and some are not. Should we unified them all?